slurm/run_eval.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash --login
+
+#SBATCH --time=00:10:00   # walltime
+#SBATCH --ntasks-per-node=8 # number of processor cores (i.e. tasks)
+#SBATCH --nodes=1   # number of nodes
+#SBATCH --mem=32G   # memory per CPU core
+#SBATCH --gres=gpu:1
+#SBATCH --qos=cs
+#SBATCH -J "lm_eval"   # job name
+#SBATCH --output=%x_%j.out
+
+# Set the max number of threads to use for programs using OpenMP. Should be <= ppn. Does nothing if the program doesn't use OpenMP.
+export OMP_NUM_THREADS=$SLURM_CPUS_ON_NODE
+
+# BEFORE RUNNING THIS SCRIPT ON THE COMPUTE NODE: Make sure to run the run_eval.py script locally to download and cache the correct datasets and benchmarks.
+# You do not need to let it run to completion, just until the datasets and benchmarks are downloaded and cached.
```

Comment:
What does this mean?

Reply:
Could downloading and caching the correct datasets and benchmarks be run earlier in the sh file?

Reply:
This will be fixed by a later pull request.